src/main/scala/edu/ie3/osmogrid/io/input/InputDataProvider.scala

```diff
@@ -92,8 +99,8 @@ object InputDataProvider extends ActorStopSupport[ProviderData] {
                 OsmReadFailed(exception)
             }
             readOsmData(providerData, replyTo)
-          case ReqAssetTypes(_) =>
-            ctx.log.info("Got request to provide asset types. But do nothing.")
+          case ReqAssetTypes(replyTo) =>
+            replyTo ! RepAssetTypes(providerData.assetSource.read())
```

Comment:
Also communicating back failures (exceptions) would also be great, similarly to it's handled with osm input data. You don't necessarily have to use futures (although one could argue that all file i/o should be executed asynchronously), but could also use a `Try`.